spec/requests/conversion_hosts_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe "ConversionHosts API" do
+  context "collections" do
+    it 'lists all conversion hosts with an appropriate role' do
+      conversion_host = FactoryGirl.create(:conversion_host)
+      api_basic_authorize(collection_action_identifier(:conversion_hosts, :read, :get))
+      get(api_conversion_hosts_url)
+
+      expected = {
+        'count'     => 1,
+        'name'      => 'conversion_hosts',
+        'resources' => [
+          hash_including('href' => api_conversion_host_url(nil, conversion_host))
+        ]
+      }
+      expect(response).to have_http_status(:ok)
+      expect(response.parsed_body).to include(expected)
```

Comment:
Should this `match` or `eq` instead?

Reply:
No, the style I used here is pretty much what all the other specs do. Neither `match` nor `eq` will work.